### DISCUSSION TITLE
How to avoid event drag and drop into past days?

### DISCUSSION THREAD

**Discussion by MarGraz** [nhn/tui.calendar#1320] [Q&A] [answered]
Hi,

I'm looking for a way to avoid the drag and drop of an event into a past date. Is there a way to do this? 
I saw that past days are marked in soft gray color, but looking in the documentation I didn't find something to avoid this behaviour.

Thank you

**Comment by adhrinae** [comment, marked as answer]
Unfortunately, we don't offer that kind of behavior.

**Reply by MarGraz** [reply]
Thank you for your answer. I will manage it showing a modal in case the user try to create an event in the past
